src/modules/household/components/elements/RelationshipToHohSelect.tsx

```diff
@@ -1,46 +1,33 @@
 import { Typography } from '@mui/material';
-import { omit } from 'lodash-es';
-import { useMemo } from 'react';
 
 import GenericSelect, {
   GenericSelectProps,
 } from '@/components/elements/input/GenericSelect';
+import { localResolvePickList } from '@/modules/form/util/formUtil';
 import { HmisEnums } from '@/types/gqlEnums';
-import { RelationshipToHoH } from '@/types/gqlTypes';
+import { PickListOption, RelationshipToHoH } from '@/types/gqlTypes';
 
 export type Option = { value: RelationshipToHoH; label: string };
 
 export interface Props
   extends Omit<GenericSelectProps<Option, false, false>, 'options' | 'value'> {
   value: RelationshipToHoH | null;
-  showDataNotCollected?: boolean;
 }
+const relationshipToHohOptions: Option[] =
+  // safe to cast because we know that RelationshipToHoH is resolvable
+  (localResolvePickList('RelationshipToHoH') as PickListOption[])
```

Comment:
do we need the cast?
```suggestion
  (localResolvePickList('RelationshipToHoH') || [])
```